azurerm/import_arm_data_lake_store_test.go

```diff
@@ -0,0 +1,54 @@
+package azurerm
```

Comment:
We are moving away from a separate import test. Lets remove this file and add the following to all the resource tests:
```
			{
				ResourceName:      resourceName,
				ImportState:       true,
				ImportStateVerify: true,
			},
```

Reply:
done, in next commit